drivers/video/video_device.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_VIDEO_DEVICE_H_
+#define ZEPHYR_INCLUDE_VIDEO_DEVICE_H_
+
+#include <zephyr/device.h>
+
+struct video_device {
+	const struct device *dev;
+	const struct device *child_dev;
```

Comment:
```suggestion
	const struct device *source_dev;
```

When links between devices are bi-directional, that blurs the limit between parent and child. Maybe `source_dev` was intended here?

Reply:
That's right. `src_dev` / `sink_dev` is better. Thanks